meal-sharing-master/src/backend/api/meals.js

```diff
@@ -82,7 +82,7 @@ HAVING max_reservation > total_reservations;
   }
 });
 
-// POST	Adds a new meal 
+// POST	Adds a new meal  
 router.post("/", async (request, response) => {
   const meals = await knex("meals").insert({
     title: "Kabab",
```

Comment:
This will always insert the same meal into the DB. You should be getting the data from the request body.

Reply:
like that right :
router.post("/", async (request, response) => {
  const meals = await knex("meals").insert({
    title: request.body.title,
    description: request.body.description,
    location: request.body.location,
    max_reservation: request.body.max_reservation,
    price: request.body.price,
    created_date: request.body.created_date,
  });
  response.json(meals);
});

Reply:
Yes, something like that where the values come from `request.body`, instead of always being the same value 👍